lib/data_update_scripts/20211206222716_remove_stackbit_page.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveStackbitPage
+    def run
+      Page.find_by(slug: "connecting-with-stackbit")&.destroy
```

Comment:
Rails 6 added `destroy_by`:

```suggestion
      Page.destroy_by(slug: "connecting-with-stackbit")
```